lib/rspecq/worker.rb

```diff
@@ -185,5 +195,23 @@ def relative_path(job)
     def elapsed(since)
       Process.clock_gettime(Process::CLOCK_MONOTONIC) - since
     end
+
+    # Prints msg to standard output and emits an event to Sentry, if the
+    # SENTRY_DSN environment variable is set.
+    def log_event(msg, level, additional={})
+      puts msg
```

Comment:
I think level and additional context should be printed as well?

Reply:
Ditto https://github.com/skroutz/rspecq/pull/16#discussion_r464918171. I'll create a new issue for this.